docker-compose.yaml

```diff
@@ -0,0 +1,12 @@
+services:
+  tinymod:
+    build: 
+      context: .
+      dockerfile: Dockerfile.dev
+      args:
+        - WORKDIR=/src/usr/tinymod
```

Comment:
is `src` and `usr` swapped here?

Reply:
🤦‍♂️ Yes they are nice catch. I am doing mkdir -p in the dockerfile so it just worked...

"Actually, as we start integrating AI models into tinymod this might be fine."
Can also bring in a database, and a admin front-end. Dunno what your long-term hosting plans are but this way you can stand up the app with "docker compose up" or if we go the route of k8s will help us there too.

Reply:
I normally use nix to manage all my servers, so i'm not too worried about the hosting end. I'm using sqlite as db rn which i think is better than having to manage a whole other db, and you can scale sqlite pretty well now with stuff like rqlite.

Dont't really want an admin frontend it is easier if everything is just done thru discord, and the new components stuff should make it work.

Reply:
Totally understandable!! My main goal is to get to a point where I can help you out with it and start hitting some of the issues your posting while being able to test. Don't want you to feel like you're going it alone that always sucks a big one when no one else helps out. If there is something else I can be doing to help you out on it  in the meantime let me know 😄

Reply:
Yea if you want to do #10, that would be great!